### DISCUSSION TITLE
Can't update a representation color theme

### DISCUSSION THREAD

**Discussion by Marwa-Da** [molstar/molstar#987] [Q&A] [answered]
Hi all,

I selected a specific chain then applied a color on the selection, from the "selection mode/ theme" window. Later, programmatically, I tried to apply `update(StateTransforms.Representation.StructureRepresentation3D` on it which wasn't successful. It still has the old color. Could you tell me how to make it work?

**Comment by dsehnal** [comment, marked as answer]
If you did it via the overpaint, that always overrides any theme. If you want to revert that, you need to query the tree (say using `state.data.selectQ`) and delete the nodes.

**Comment by Marwa-Da** [comment]
What is the primary purpose of overpaint? Why not utilize the theme instead? I'm quite curious!

**Reply by arose** [reply]
to let users easily paint over any existing color theme

in many cases a theme can indeed be used, but overpaint offers a second path for coloring that does not need to know about any themes, exiting or future ones

**Reply by Marwa-Da** [reply]
Thank you both so much!
